aurora/core.py

```diff
@@ -123,17 +128,87 @@ def reload_namelist(self, namelist=None):
             self.namelist = namelist
 
         # Extract other inputs from namelist:
+            
         self.mixing_radius = self.namelist["saw_model"]["rmix"]
         self.decay_length_boundary = self.namelist["SOL_decay"]
-        self.wall_recycling = self.namelist["wall_recycling"]
+        
+        # Edge parameters
+        
+        self.screening_eff = self.namelist["screening_eff"]
+        if self.screening_eff < 0.0 or self.screening_eff > 1.0:
+            raise ValueError("screening_eff must be between 0.0 and 1.0!") 
+        self.div_recomb_ratio = self.namelist["div_recomb_ratio"]
+        if self.div_recomb_ratio < 0.0 or self.div_recomb_ratio > 1.0:
+            raise ValueError("div_recomb_ratio must be between 0.0 and 1.0!") 
         self.tau_div_SOL_ms = self.namelist["tau_div_SOL_ms"]
-        self.tau_pump_ms = self.namelist["tau_pump_ms"]
+        
+        # Plasma-wall interaction parameters
+        
+        self.wall_recycling = self.namelist["wall_recycling"]
```

Comment:
Actually, I have thought a bit more about it, and I thing it might make sense... it will cost 30/40 list at the beginning of core.py to unwrap all the variables, but later in the various methods and functions the lines will be more slim and more easily readable, as we will have e.g. `self.wall_recycling` rather than `self.namelist["wall_recycling"]`...